src/AbstractZohoDao.php

```diff
@@ -529,15 +529,31 @@ public function updateRecords(array $beans, $wfTrigger = null)
      * Implements uploadFile API method.
      *
      * @param string $id      Zoho Id of the record to retrieve
-     * @param string $content The string containing the file
+     * @param resource|string $content A handle, or the inlined content of the file
+     * @param string $filename The string containing the name of the file saved on Zoho CRM
      *
      * @return Response The Response object
      *
      * @throws ZohoCRMResponseException
      */
-    public function uploadFile($id, $content)
+    public function uploadFile($id, $content, $filename)
```

Comment:
In order to not break compatibility with previous releases, could you simply put the $filename parameter as optional?

```
public function uploadFile($id, $content, $filename = null)
```